pkg/event/bus.go

```diff
@@ -20,7 +20,8 @@ type Hook struct {
 }
 
 type event struct {
-	Name string
+	Name    string
+	Success bool
```

Comment:
Hey! Thanks a lot for the PR.

After thinking for a while, I think this can better be `Error error` so that it can be used like `{{ if not .Error }}...{{ else }}...{{ end }}` which seems more idiomatic to users with some golang background.

It may also open up more possibilities like using hooks for error reporting to Slack/GitHub/MS Teams/etc, as you now have access to the error caused the installation to fail.

WDYT?

Reply:
Oh wow I really like that, it heaps better! I'll make those changes when I have time 🚀